recipes/dev/lora_finetune_fsdp2.py

```diff
@@ -230,6 +231,10 @@ def setup(self, cfg: DictConfig) -> None:
         )
 
         self._loss_fn = config.instantiate(cfg.loss)
+        if self._model_compile:
+            log.info("Compiling loss with torch.compile...")
+            backend = os.environ.get("TORCH_COMPILE_BACKEND", "inductor")
+            self._loss_fn = torch.compile(self._loss_fn, backend=backend)
```

Comment:
lets leave the loss compile out of this recipe. I can add it to the chunked CE, since it needs some extra logic.